package.json

```diff
@@ -21,27 +25,42 @@
   },
   "homepage": "https://github.com/fmauquie/react-angular#readme",
   "devDependencies": {
-    "angular": "^1.5.8",
     "angular-mocks": "^1.5.8",
     "babel-cli": "^6.11.4",
+    "babel-eslint": "^9.0.0",
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.11.1",
     "babel-preset-stage-2": "^6.11.0",
     "chai": "^3.5.0",
+    "enzyme": "^3.6.0",
+    "enzyme-adapter-react-16": "^1.5.0",
+    "eslint": "^5.6.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jest": "^21.22.0",
+    "eslint-plugin-jsx-a11y": "^6.1.1",
+    "eslint-plugin-react": "^7.11.1",
+    "http-server": "^0.11.1",
```

Comment:
`http-server` isn't required to build or test right? Assume it's a global install